core/common/src/main/java/alluxio/AlluxioURI.java

```diff
@@ -237,6 +229,20 @@ public String getName() {
     return path.substring(slash + 1);
   }
 
+  /**
+   * Gets the host of the {@link AlluxioURI}.
```

Comment:
NIT: it is not clear from the javadoc that it returns `null` for all authorities rather than `SingleMasterAuthority`.